### DISCUSSION TITLE
Random Mod Improvements

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#21684] [Ideas]
It would be nice if you could view the difficulty a map will have on Random before playing it (since afaik it's only knowable after you've played the map), and if there were a way of seeing the current seed and changing it with a bind (maybe just display the seed and new difficulty every time you press the bind). 

This would be nice so that I don't accidentally snipe NM scores with a Random Mod score simply because I got an easier seed that lowered the difficulty, which I wouldn't know until I had finished the map.

**Comment by mk56-spn** [comment]
as per this comment https://github.com/ppy/osu/discussions/20926#discussioncomment-3960596 previews for some mods will  probably be a thing, im too busy to tackle it myself but the "gameplay preview" im using for practice mode https://github.com/ppy/osu/pull/21408 would work since it is capable of showing gameplay  in menus,  will probably require a bit of effort to implement cleanly into the mod select overlay though
